src/main/java/io/fabric8/maven/docker/StartMojo.java

```diff
@@ -331,6 +336,26 @@ public StartedContainer call() throws Exception {
         return imagesWaitingToStart;
     }
 
+    private void createVolumesAsPerVolumeBinds(ServiceHub hub, List<String> volumeBinds, List<VolumeConfiguration> volumesConfigs)
+        throws DockerAccessException {
+        Map<String, Integer> aIndexMap = new HashMap();
+        for(Integer aIndex = 0; aIndex < volumesConfigs.size(); aIndex++)
+            aIndexMap.put(volumesConfigs.get(aIndex).getName(), aIndex);
+
+        for(String aVolumeBind : volumeBinds) {
+            if(aVolumeBind.contains(":")) {
+                aVolumeBind = aVolumeBind.substring(0, aVolumeBind.indexOf(":"));
+            }
+            Integer nVolumeConfigIndex = aIndexMap.get(aVolumeBind);
```

Comment:
What's the difference between an `n` and a `a` prefix for your variables names ? tbh, I would avoid that kind of prefixing as we don't do it nowhere else in this code base. It's always good to adapt to a given codestyle instead of introducing ones own (like avoid braces or spaces after `for` or `if`). This helps to keep the code consistently readable.